.gitignore

```diff
@@ -2,3 +2,8 @@
 *.a
 *.o
 *.so
+
+build/
+install/
+log/
+
```

Comment:
These look like the folders that are created by colcon in the workspace. Why do we need to add them to the package gitignore? They should be outside of the source tree.

Reply:
You are right. I know see that those were created here because I mistakenly started the build from the package folder instead of the workspace folder.